src/parser/jobs/ast/modules/ArcanaTracking/ArcanaTracking.tsx

```diff
@@ -91,45 +82,16 @@ export default class ArcanaTracking extends Analyser {
 	private on_prepullArcanas?: Array<Events['statusApply']>
 	private off_prepullArcanas?: Array<Events['statusRemove']>
 
-	private astrosignGauge = new EnumGauge({
-		maximum: 3,
-		options: [
-			{
-				value: SealType[SealType.CELESTIAL],
-				label: <Trans id="ast.arcana.astrosigns.celestial">Celestial</Trans>,
-				color: CELESTIAL_COLOR,
-				tooltipHideWhenEmpty: true,
-			},
-			{
-				value: SealType[SealType.SOLAR],
-				label: <Trans id="ast.arcana.astrosigns.solar">Solar</Trans>,
-				color: SOLAR_COLOR,
-				tooltipHideWhenEmpty: true,
-			},
-			{
-				value: SealType[SealType.LUNAR],
-				label: <Trans id="ast.arcana.astrosigns.lunar">Lunar</Trans>,
-				color: LUNAR_COLOR,
-				tooltipHideWhenEmpty: true,
-			},
-		],
-		graph: {
-			handle: 'astrosigns',
-			label: <Trans id="ast.arcana.astrosigns.label">Astrosigns</Trans>,
-			tooltipHideMaximum: true,
-		},
-	})
-
 	override initialise() {
 		// Initialize grouped reference to actions/statuses data
-		this.play = PLAY.map(actionKey => this.data.actions[actionKey].id)
-		this.arcanaStatuses = ARCANA_STATUSES.map(statusKey => this.data.statuses[statusKey].id)
+		this.play = PLAY_I.map(actionKey => this.data.actions[actionKey].id)
+		this.arcanaStatuses = DAMAGE_INCREASE_STATUS.map(statusKey => this.data.statuses[statusKey].id)
 		this.cardGrantingAbilities = CARD_GRANTING_ABILITIES.map(actionKey => this.data.actions[actionKey].id)
 		this.cardActions = CARD_ACTIONS.map(actionKey => this.data.actions[actionKey].id)
 		this.drawnArcana = DRAWN_ARCANA.map(statusKey => this.data.statuses[statusKey].id)
-		this.celestialSealArcana = CELESTIAL_SEAL_ARCANA.map(actionKey => this.data.actions[actionKey].id)
-		this.lunarSealArcana = LUNAR_SEAL_ARCANA.map(actionKey => this.data.actions[actionKey].id)
-		this.solarSealArcana = SOLAR_SEAL_ARCANA.map(actionKey => this.data.actions[actionKey].id)
+		this.celestialSealArcana = DAMAGE_INCREASE_ARCANA.map(actionKey => this.data.actions[actionKey].id)
+		this.lunarSealArcana = HEAL_MIT_ARCANA.map(actionKey => this.data.actions[actionKey].id)
+		this.solarSealArcana = REGEN_SHIELD_ARCANA.map(actionKey => this.data.actions[actionKey].id)
```

Comment:
Is this the reason why the arcana had to be split like that? If so then maybe you can ignore my comment about the group and just leave it as it is. Seems like it would be a pain and should be handled in a later PR.

Reply:
I think the only reason I can see to split them up is if we display them on the arcana logs and want to synth any precasts (but I guess we can also just check for any of the actions). I think another reason I wanted to split them was to demonstrate that they are different and hopefully would get people thinking about what they do differently, but I guess it doesn't wholly matter since they will still have to do that within each play_II and play_III anyway and hopefully they are remembering which cards do what.

I don't see any other reason to separate them at this time
Let me get back to you on this (I'll probably combine unless we have a specific purpose in the future but that person can split them in a way that works for them lol)

Thanks for the feedback

Reply:
Updated as of commit 3b4f7a9 below
we'll definitely have a bit of work in this module to get it to do what we want at least